### PR TITLE
NAPD MWL 2.0 support

### PR DESCRIPTION
**Pull request by jwarwick**
Parsing the new MWL json format from NRDB. Support for banned and restricted cards. Removed universal influence code.

Currently reading from a local json file as NRDB API doesn't support the new format yet.
This is the issue on the NRDB side which needs to be committed: https://github.com/Alsciende/netrunner-cards-json/pull/258

### CODE REVIEW

**Comment by danhut** [comment]
All my hard work on MWL gone thanks to Boggs 👍 
Good work on this!

File: src/cljs/netrunner/deckbuilder.cljs [hunk 1]
```diff
@@ -508,7 +499,8 @@
 ;; Dot definitions
 (def zws "\u200B")                                          ; zero-width space for wrapping dots
 (def influence-dot (str "●" zws))                           ; normal influence dot
-(def mwl-dot (str "★" zws))                                 ; influence penalty from MWL
+(def banned-dot (str "✘" zws))                              ; on the banned list
+(def restricted-dot (str "🦄" zws))                          ; on the restricted list
```
Comment (Saintis) [inline src/cljs/netrunner/deckbuilder.cljs]: Love it!

Comment (jwarwick) [reply src/cljs/netrunner/deckbuilder.cljs]: I can't take credit - I think I saw it used somewhere on Reddit first :)

**Comment by mtgred** [comment]
Awesome 👍

**Pull request merged**